apps/privacy-panel/locales/privacypanel.en-US.properties

```diff
@@ -88,24 +94,26 @@ passphrase-invalid = You cannot use special characters!
 passphrase-different = Confirmation must match passphrase!
 pin-empty = Passcode lock is empty!
 pin-invalid = Wrong Passcode lock!
+passcode-long = Passcode should be 4 digits!
 pin-different = Wrong Passcode lock!
-sim-invalid = Wrong SIM PIN!
+sim-invalid-txt = Wrong SIM PIN! {{ value }} trie{{ plural }} left!
```

Comment:
What exactly is this?? [Plural forms](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Use_proper_plural_forms).